app/api/orders.py

```diff
@@ -139,7 +139,7 @@ def after_create_object(self, order, data, view_kwargs):
 #             TicketingManager.calculate_update_amount(order)
 
         # send e-mail and notifications if the order status is completed
-        if order.status == 'completed':
+        if order.status == 'completed' or order.status == 'placed':
```

Comment:
@shreyanshdwivedi Your logic on FE and here don't agree, there after setting the status of order as placed, you are asking for user to fill payment details, where as this would actually send the tickets to him. 
https://github.com/fossasia/open-event-frontend/pull/2955/files#r285388216

Reply:
I'll be updating front-end logic and display a different success message.